Source/ACE/Entity/WorldObject.cs

```diff
@@ -699,8 +730,61 @@ public void WriteUpdatePositionPayload(BinaryWriter writer)
             writer.Write(Sequences.GetCurrentSequence(SequenceType.ObjectForcePosition));
         }
 
-        public virtual void Tick(double tickTime)
+        protected void ForceUpdatePosition(Position newPosition)
+        {
+            ForcedLocation = newPosition;
+        }
+
+        /// FIXME(ddevec): We're probably ultimately only going to update velocity, not true position...
```

Comment:
we'll have to do both.  velocity for client prediction, position for client correction.